smoke-test/src/test/java/org/opennms/smoketest/topo/GraphMLTopologyIT.java

```diff
@@ -238,9 +238,25 @@ public void verifyCanFilterByCategory() throws IOException, InterruptedException
         Assert.assertEquals("Servers", topologyUIPage.getVisibleVertices().get(1).getLabel());
     }
 
-    /**
-     * Creates and publishes a requisition with 2 dummy nodes with predefined parameters
-     */
+    @Test
+    public void verifyCanChangeIcon() throws IOException, InterruptedException {
+        topologyUIPage.selectTopologyProvider(() -> LABEL);
+        topologyUIPage.findVertex("North Region").contextMenu().click("Navigate To", "Markets (North Region)");
+        String label = "North 1";
+
+        // open icon select popup
+        topologyUIPage.findVertex(label).select();
+        topologyUIPage.findVertex(label).contextMenu().click("Change Icon");
+
+        // choose new icon
+        String newIconName = "microwave_backhaul_1";
+        findElementByXpath("//*[name()='title' and text()='"+newIconName+"']/../*[name()='rect']").click();
```

Comment:
Can we move this to a `setIcon(...)` method?

Reply:
to make sure: you want me to put the whole process of choosing an icon in the method?
That would be:
* click on an Element
* open the context menu via right click
* open the change icon dialog
* select an icon
* close the icon dialog

Reply:
Hmm sorry, the comment was in state `pending`, but I wrote it 4 days ago, is it visible to you now?

Reply:
now it is :-) 
the code snipped you posted does steps 1-3 but I assume we also want step 4. I'll wrap it in a method.

Reply:
Yeah, makes sense (-: